docs/modules/ROOT/pages/vshn-managed/keycloak/customization.adoc

```diff
@@ -0,0 +1,28 @@
+= Customization
+
+This page will describe how you can customize your Keycloak instance.
+
+== Themes and providers
+
+`VSHNKeycloak` allows to use custom providers and themes.
+xref:vshn-managed/how-tos/custom-image-keycloak.adoc[Create] a custom container image and use it to customize your Keycloak instance as described below.
+
+.Example to create a Keycloak instance an image containing custom themes and providers
+[source,yaml]
+----
+apiVersion: vshn.appcat.vshn.io/v1
+kind: VSHNKeycloak
+metadata:
+  name: keycloak-app1-prod
+  namespace: prod-app
+spec:
+  parameters:
+    service:
+      version: "23"
+      customizationImage: https://registry/user/image:tag # <1>
```

Comment:
This is wrong. The `customizationImage` is a type that has the subkeys  `image` for the actual image as well as `imagePullSecretRef` for the optional pullSecret (if that is required)

Reply:
oh thanks for the note, I will adjust the code